File: src/pocketcloud/cloud/command/impl/ConfigureCommand.php
<?php

namespace pocketcloud\cloud\command\impl;

use pocketcloud\cloud\command\Command;
use pocketcloud\cloud\command\sender\ICommandSender;
use pocketcloud\cloud\setup\impl\ConfigSetup;

final class ConfigureCommand extends Command {

    public function __construct() {
        parent::__construct("configure", "Reconfigure the config");
    }

    public function run(ICommandSender $sender, string $label, array $args): bool {
        new ConfigSetup()->startSetup();
        return true;
    }
}

File: cloud-core/src/main/java/de/pocketcloud/cloud/console/command/impl/ConfigureCommand.java
package de.pocketcloud.cloud.console.command.impl;

import de.pocketcloud.cloud.console.command.Command;
import de.pocketcloud.cloud.console.command.ctx.CommandContext;
import de.pocketcloud.cloud.console.command.desc.CommandDescription;
import de.pocketcloud.cloud.console.command.sender.CommandSender;
import de.pocketcloud.cloud.setup.def.ConfigSetup;

@CommandDescription(name = "configure", description = "Reconfigure the cloud", aliases = {"reconfigure", "reconf", "conf"})
public final class ConfigureCommand extends Command {

    @Override
    public void prepare() {}

    @Override
    public boolean run(CommandSender sender, CommandContext ctx) {
        new ConfigSetup().startSetup();
        return true;
    }
}